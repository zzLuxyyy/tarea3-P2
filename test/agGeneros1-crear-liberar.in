crearAGGeneros
liberarAGGeneros
crearAGGeneros
liberarAGGeneros
Fin