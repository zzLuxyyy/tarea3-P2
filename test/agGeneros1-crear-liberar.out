1>Se ejecut贸 'crearTAGGeneros' exitosamente.
2>Se ejecut贸 'liberarTAGGeneros' exitosamente.
3>Se ejecut贸 'crearTAGGeneros' exitosamente.
4>Se ejecut贸 'liberarTAGGeneros' exitosamente.
5>Fin.
